### DISCUSSION TITLE
阿里云函数计算部署 docs文档标题错误

### DISCUSSION THREAD

**Discussion by musnows** [walinejs/waline#1742] [General]
如题，这里的修改环境变量和更新版本的标题是不是错了？

https://waline.js.org/guide/deploy/aliyun-fc.html#%E5%A6%82%E4%BD%95%E6%9B%B4%E6%96%B0

![image](https://user-images.githubusercontent.com/87506951/212306730-bdcd2466-a723-4bc9-afae-041b19c2e85d.png)

**Comment by Mister-Hope** [comment]
PR welcome
